specification/toolingservice/data-plane/1.0/toolingservice.json

```diff
@@ -0,0 +1,755 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tooling Service Client",
+    "description": "Client that can be used to access Tooling services to run linter, static and api validations.",
+    "version": "1.0"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "host": "<TBD>",
+  "basePath": "/api/",
+  "schemes": [
+    "https", "http"
+  ],
+  "paths": {
+    "/tools/lint": {
+      "post": {
+        "tags": [
+          "Linter"
+        ],
+        "operationId": "Tools_Linter",
+        "description": "Get Linter Result",
+        "parameters": [
+          {
+            "name":"repository",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Name of the repository"
+          },
+          {
+            "name":"branch",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Name of the specific branch in the repository."
+          },
+          {
+            "name":"configPath",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Configuration path of the file."
+          },
+          {
+            "name":"apiTag",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "API Tag to be validated."
+          },
+          {
+            "name":"commitHash",
+            "in": "body",
+            "type": "string",
+            "description": "Commit hash to be validated."
+          },
+          {
+            "name":"mode",
+            "in": "query",
+            "type": "string",
+            "description": "Mode"
+          }          
+        ],
+        "responses": {
+          "400": {
```

Comment:
documenting error codes may result in SDK behavior that we don't want, please check on this.

Reply:
Reference: https://github.com/Azure/azure-rest-api-specs/blob/32b0d873aa851d456dfde7d6ba1d89ff33f897f0/documentation/creating-swagger.md#negative-responses

Some proposal for improvement on this https://github.com/Azure/autorest/blob/bc0dcde0a05606a1b102c0e08c1798fa4da4dcc8/docs/proposals/generator-specific-settings/improved-exceptions.md